controllers/get-reviews-id-controller.js

```diff
@@ -0,0 +1,15 @@
+const reviews = require('../db/data/test-data/reviews');
+const {fetchReviewsByID} = require('../models/get-reviews-id-models');
+
+function getReviewsByID(require, response, next){
+    const {review_id} = require.params;
+    fetchReviewsByID(review_id)
+    .then((reviews) => {
+     response.status(200).send({reviews: reviews})
```

Comment:
As this is a single review, we should send it back on a key of review, not reviews.